etc/kayobe/hooks/overcloud-host-configure/post.d/99-cis.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+- import_playbook: ../../../ansible/cis.yml
+  when: stackhpc_enable_cis_benchmark_hardening | bool
```

Comment:
Also it's not clear from the name that this only affects the hook.

Reply:
Fair point. I will move it into the playbook.

Reply:
I changed my mind as that is technically a breaking change and updated the name of the variable.